templates/company/view.html.tx

```diff
@@ -375,10 +406,17 @@
             </table>
     %    }
     % }
-    % if $company.foreign_company_details {
+    % if $company.foreign_company_details || $company.type {
```

Comment:
Should this be `if $company.foreign_company_details || $company.type == "registered-overseas-entity"`